projects/js-packages/charts/src/components/shared/with-responsive.tsx

```diff
@@ -6,10 +6,11 @@ type ResponsiveConfig = {
 	maxWidth?: number;
 	aspectRatio?: number;
 	debounceTime?: number;
+	useSingleDimension?: boolean;
```

Comment:
is this property in use? @annacmc

Reply:
Ah no - this is an artifact from an approach instead of using size, to allow it to use either size or height + width. But it's not the best way to do this. I will remove in followup PR also, thanks 🙏